src/lang/TranslationsRU.js

```diff
@@ -0,0 +1,142 @@
+export default {
+    ru: {
+        Home: {
+            record: 'Рекорд',
+            historyBtn: 'ИСТОРИЯ',
+            about: 'ПРО ПРОЕКТ',
+            searchBar: {
+                customLoaded: 'Пользовательская карта загружена',
+                enterCity: 'Введите страну, штат, город',
+            },
+            aboutDescriptions: {
+                1: 'Geoguess это бесплатная и ленивая игра без рекламы. \
+                Игроки соревнуются, насколько близко игрок может угадывать случайные места за пять раундов. \
+                Вы можете поделиться счетом с другими людьми через социальные сети, такие как Facebook или Twitter. \
+                Вы можете играть в многопользовательскую игру со своими друзьями до пяти друзей. \
+                Первый игрок создает комнату и определяет ее размер. Другие игроки вводят то же имя комнаты, что и первый созданный игрок, и игра начнется. ',
+                2: 'Эта игра была разветвлена ​​от GeoGuess Master ',
+            },
+            customMap: 'CUSTOMS MAP',
```

Comment:
@BurAndBY Can you translate this string ? :)

Reply:
Oh yeah thanks